packages/vaadin-dialog/theme/lumo/vaadin-dialog-styles.js

```diff
@@ -18,6 +18,7 @@ registerStyles(
     }
 
     [part='overlay'] {
+      border-radius: var(--lumo-border-radius-l);
```

Comment:
This appears to be a breaking change that affected screenshots. Should we split this to separate PR?

I have updated the PR title to add `refactor!` indicating a breaking change for now.